src/Model/FocusManager.re

```diff
@@ -21,6 +23,8 @@ let pop = (focusable: Focus.focusable, state: State.t) =>
 let current = (state: State.t) =>
   if (Sneak.isActive(state.sneak)) {
     Focus.Sneak;
+  } else if (state.modal != None) {
+    Focus.Modal;
```

Comment:
The modal focus is now also enforced by the presence of a modal, rather than being manually controlled.

Reply:
Nice!